src/solvers/dgsem_tree/indicators.jl

```diff
@@ -212,6 +212,85 @@ end
 
 
 
+"""
+    IndicatorIDP(equations::AbstractEquations, basis;
+                 IDPPositivity=false,
+                 variables_cons=(first,),
+                 positCorrFactor=0.1)
+
+Blending indicator used for subcell shock-capturing [`VolumeIntegralShockCapturingSubcell`](@ref) including:
+- positivity limiting for conservative variables.
+
+## References
+
+- Rueda-Ramírez, Pazner, Gassner (2022)
+  "Subcell Limiting Strategies for Discontinuous Galerkin Spectral Element Methods"
+- Pazner (2020)
+  "Sparse invariant domain preserving discontinuous Galerkin methods with subcell convex limiting"
+  [arXiv:2004.08503](https://doi.org/10.1016/j.cma.2021.113876)
+
+!!! warning "Experimental implementation"
+    This is an experimental feature and may change in future releases.
+"""
+struct IndicatorIDP{RealT<:Real, LimitingVariablesCons, Cache} <: AbstractIndicator
+  IDPPositivity::Bool
```

Comment:
```suggestion
  idp_positivity::Bool
```
For me as a developer, `IDPPositivity` really reads like a type, because we only use `CamelCase` for types. If there are no other strong reasons, I'd really prefer to keep using `snake_case` for variables (and functions).

Will there be other positivity-type variants added later? if not, the `idp` is redundant and you could just call it `positivity` (since the type already has `IDP` in its name)